mypy/errorcodes.py

```diff
@@ -301,6 +301,9 @@ def __hash__(self) -> int:
 POSSIBLE_FUNCTION: Final[ErrorCode] = ErrorCode(
     "possible-function", "possible FunctionType on class", "General", default_enabled=False
 )
+ERM_WHAT: Final[ErrorCode] = ErrorCode(
+    "unhandled-scenario", "don't know what happened here...", "General", default_enabled=False
```

Comment:
it should probably say something more descriptive like "an unknown error occurred. raise an issue" or something

Reply:
to be fair, that text never shows anywhere...

Reply:
ive written error messages before which i also expected to never happen and was often wrong. i think if theres any chance at all the user could see this message it should say that it was some sort of internal/unknown error

Reply:
I mean, this second parameter `description` is never referenced in the code base at all